### PR TITLE
Add new GIFs

### PR DESCRIPTION
**Pull request by eilla1**
I added new GIFs for demonstrating a pull request to reflect the new GitHub interface.

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -111,10 +111,10 @@ A commit message is used for quickly summarizing a change. Another contributor s
 
 Here are a series of GIFs that illustrate how to make and commit a change, and create a pull request for review, using GitHub's web interface.
 
-![](img/gh_web_find_file.gif)
+![](https://cloud-agvs502ft.vercel.app/0pr1.gif)
```
Comment (sampoder) [inline CONTRIBUTING.md:114]: Could we add alt text on these all?

**Review by sampoder — CHANGES_REQUESTED** [review]
Thank you for doing this! Quick ask:

File: CONTRIBUTING.md (hunk 1 above)
Comment (eilla1) [reply CONTRIBUTING.md]: Done! Thanks for pointing that out :)

**Review by KhushrajRathod — APPROVED** [review]

**Review by sampoder — APPROVED** [review]
thanks for doing this!

**Pull request merged**